src/embed.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+window.addEventListener("DOMContentLoaded", (event) => {
+    // Set nice style defaults for embedded diagrams.
+    for (embed of document.querySelectorAll(".quiver-embed iframe")) {
+        embed.style.border = "none";
+        embed.style.borderRadius = "5px";
+        embed.style.overflow = "hidden";
```

Comment:
When is it possible for overflow to happen without this?

Reply:
Ah... you're definitely right. No it's not.